test/Microsoft.AspNet.Mvc.IntegrationTests/MutableObjectModelBinderIntegrationTest.cs

```diff
@@ -268,9 +268,6 @@ private class Order2
         private class Person2
         {
             public string Name { get; set; }
-
-            [FromServices]
-            public IActionBindingContextAccessor BindingContext { get; set; }
         }
 
         [Fact]
```

Comment:
This test isn't valid any more. This should be updated to use `[FromBody]` or something similar. If `[FromBody]` is already covered then this is good enough.